support/build.gradle

```diff
@@ -37,17 +37,9 @@ if (JavaVersion.current().isJava8Compatible()) {
     }
 }
 
-// todo: will update this lib to sampingantech jcenter
-publish {
-    userOrg = 'nikart'
-    groupId = 'com.github.nikartm'
-    artifactId = 'image-support'
-    publishVersion = '1.1.0'
-    desc = 'Library to add ImageView (ImageBadgeView) with a badge like notification count'
-    licences = ['Apache-2.0']
-    uploadName='ImageBadgeView'
-    website = 'https://github.com/nikartm/Image-Support'
-}
+group = "com.github.sampingantech"
+version = android.defaultConfig.versionName
+archivesBaseName = 'image-support'
```

Comment:
looks like when using this, I think we don't have to use capitalize `Image-Support` anymore

because jitpack will runs this task:

```
./gradlew clean -Pgroup=<overrided from support/build.gradle> -Pversion=<overrided from support/build.gradle> -xtest -xlint install
```

by default the task will search for `archiveBaseName` as stated from this docs: https://docs.gradle.org/current/userguide/base_plugin.html#sec:base_plugin_conventions

if `archiveBaseName` null 
else check (if defined) `$project.name` or `rootProject.name` then use it
else it'll use repository name, e.g: `Image-Support`

assign `archiveBaseName` is a common usecase to define an `artifactId`, as stated in this formula:

```
<groupId>:<artifcatId>:<versionName>

com.github.sampingantech:material-calendarview:x.y.z
```

cmiiw or you can try to run this locally: `./gradlew :support:publishToMavenLocal`

@anangkur

Reply:
Ooops my bad, I've reread the jitpack docs, the `archiveBaseName` wouldn't do nothing and still using repository name isntead:

https://jitpack.io/docs/#building-with-jitpack:~:text=Add%20the%20dependency%20information

Reply:
alright mas, so we still using Image-Support @mochadwi .

Reply:
Yes, but it's safe to try the above task locally to make it's successfully published to maven local

Reply:
alright mas will try @mochadwi